tests/multi_net/ssl_cert_rsa.py

```diff
@@ -0,0 +1,45 @@
00\xd0^\xd6\xab\x9b\x9f\xbd\xf7\xa0\xf5\xf9\x1fh\xdc;\xacW\x893-n\xceF\x10?\xb1\xef\x10\x9f\xc9r\xfd\xc9\x9e\xdf1\x07\xa2=f\xd1\xcd\xfek\xdd\xdf\xd1\xbb9R\xcc\xd1\x0b\\ \xad\x1b>\n\xa6\xa5\x12q\xec\xf3\xa7\xef*e\xe0)\xf5\xd7\x7f#\x8d\x125\xb5*\x9d\xca4Q\xc1e\xd7\n\x99\xcb\xae\xa5\xc6\x10\xe5EYyim\xb7i\x19\x1e|\xf2\xdb!\xf6A\x95\x9eK\xa1\xc5\xc0\xaa\xe2`\xc7$\x96+j\xc2b\x1d\x92\xe9\xdfz\xde\xb8+R-7\xb4,\xb4T\x00;\xbe`\xd9\x91[\xf7sz\xec\xcf\x88\xc7\xed\x12c\xa2/C\x1asNa\xfeqs\xa97\xdd\xf7j\xd2\xa7\x99\x94\xc0R8\xde\xfc\x15\xf6\x84hX\xe9\xed\xf2z\xe2\xa5g\xc7\xc5\xc75\xea]/\xbe\xf6Z!\x95\xbc\x05\xd8,\xbf\x06\xa4w\xb2\x9c\x84\xc9.\x80T\xc2\xbb%\xd8\xc6\xf1\x9dC\xef_\xd1\xfc\xe1<,\xdb\xc3a\xc3\x9b\xae\xc3{9\x92\x00\xb3-Jg\x98\xba\x0bTa\x02\x82\x01\x01\x00\x8eAI,N}\xaf\xf76\x8d\x1dld\x03@g\xa9M\xcaTa\xa00\x1e \x1a\xdd.\r\\\xcb\x8c\xb45h[\xfa\x98\xe3bW,\xf8#j\x10\xd1\x91\xb1\x87\xa5h\xae\xe6\x88\xb6\xc6\x00P\xd1\xbc\x18\x1d\x7f\xd5|\x863\x19[\xf5\xb7Wkc|o\xb3X\xda\xe8\xb5,\xcc\x15\x81Z\xff\xb9\x9e3A7\xdc\xb9\x1a\x834u\xdb/@\x04\x16K] ,l\x1b\xbf\tJP\xdc~p\xec\x9f\x0e\xd0g\xbbiD\xb1\xe7\xe3\xc8\x97\xaa\xec\xfcS\x98J\xdd\x1cO\xf5\xb5%\x03L\xf3\xca\x95\xe8\xa0\x9a\xeb\xa8\x04\xf1\xc7\xe0+\xe3\x91\xb2\xbcd\x11f\xc3\xe6T\xee\xf5\xe7-\xba7\xd9\x8f@o?\xa5 \xe4\x1f.\xa1\x0fUt\xacY\x84\xf7QE7\x8f\xef\xbf\xac\x1d\x07\xff\xf3\xf24\xfe\xc6\x98\xd5^tk\x1d\xa1\x8fo}\xe2N\xc8N\xd7\xcbDmB\x88 \xbe\xf3<\x00i>j\x0e\xf1\x14\xb5\xd6n\x9f\xef\xa8\xee\x05\x928\xdf\x1a\xc3|\x87\xe7\x84\x1a\xe7\x02\x82\x01\x00\x07!\xa7\xd19\xc3N\xda!\xcd)X\x94\xdb,\xc3\xaa?L\xdc\x1a5\xbf\x1a!C\xf2\xbd\xab\xeaV /}[\x80/\x15\xb3jHu\xf7f3\xb2\xccW\xcf\x0fqi\x1a-n\x04\xde\xb0\xd1\xe6\x801\xd0j^\xb0y\xb4\x06\xc6\x94I\x10\xb6\x0e>n\xc8\x1d\xca6\x9aL\x0e\x1cCc\x07\xbe\xd9\xc4\xc1q\xb4\xf4S\xdaK\x18{\xa3\xd2Z\x04\xbc\x1c\x07\xb9\xf2\xd6\xad\xcb<%nB8\xd7\x04\x9e\xec6\xa3\x87\xc4\xdd\\\xcb\xc1k_\xa6-\xc1u\xcf\x8c_\x83D,\xb7\xd1S\xa3\xb6\xee\x8d\xaa;n\x92\x9aK\xc1#\xf1\x04-\xf1\xd6'\x1a\x99-+k0\x9d3\x07J\xc7\x82,0Jr\x06\x9ea\xabY\t\x15\xe1\x08b\x01=\xd2L\xdd\x82^\xc8\xfb\x17rL\xfc,Y\xfc\x1d\xb8%6A\xfe\xce\x0e\xe9$\x1b\x9d\xd5\xc1\x98\xf0\xd5u\xd0\xb7\xeb\xe2k4\x89\xb5\xb0\x9e\xdc;\xcd6o\xd3\x11\x0e\x83\xce\x88l8=1\xfe\xef\xe6\xe3\x02\xcc#E!\x02\x82\x01\x00\x08\xf7z3\xd0\x08\x1e\x9b\xe3\xc1\xb1\xac\x8b\x8e\x0e\xeb\xb7-\xf2\xebi\xb9].\xd7I5\xb9\xda\xb8\xe1p#\xcc8FST\x02<Q\x83\xb5\x1aj (\x8f\xbb!\x81\x17+\xe1\xc2\xfd\xb8\xb4DA\x94T\xe5\xb3\x7f\x7f;\xdf\x11\xe2\x8c\xf4tk%SN\xb6/~\x87\xbb\xbf(\xed\xa3p$6\x8b8\x97\xfb\xc6a\xb4\n\x93\xe0J\x18=\xb9!\x9c\x04\xa8vC\xed\xf5\xd8\xb5\xdb\xfe=BN\x91\xd5X\xd5\xe3\xe2\xfa\x02\xce\x19\x84\xeei\xfb\x85\x18G\x0e\xee.}\xb0\xe1\xdfZ\xc4W\x1fx\xa7\xa2R\x9b\xc1\xfe\xf5\xe3-F\x99Hi\xa8\xd8\xccG\x86\x9e\x17M\x84\xe7\x97k\xe8\xeb\xb8\x8f,\xcbq\xa6\x03\xa8\xbd\xb0j\xf3\xeb-\xdb\xd6 \x82\xf4\ry\x87\xe4\x7f.2\x1e\xb5\xeb*(\xfe\xfa\xb2c@\x9f\x89\xdc\x97\xeb\xc7#\xa1\xb7QA\x8c\xdd>\xa6\x84\xba\x8b\x17\xa30\xa3\x06\xa6\xfb\xcfQ\xba\x83V:\xed\x85\xa4\xf8\x86\xff\xf1\xa2$#t\x8d\x83y\x8c"
```

Comment:
Please format these over multiple lines, using unhexlify like it was done in `ssl_data.py`.

Reply:
Sure, I will add `server_hostname` to docs too.